lms/static/sass/elements/_course-card.scss

```diff
@@ -1,8 +1,46 @@
-.header-img{
-  @include float(left);
-  margin-right:20px;
-}
 .course-card{
-  height: 250px;
-  margin-top:20px;
+  width: 240px;
+  @include susy-media($bp-screen-sm) { width: 240px; }
+  @include susy-media($bp-screen-md) { width: 700px; }
+  @include susy-media($bp-screen-lg) { width: 920px; }
```

Comment:
@AlasdairSwan I don't really understand what changes I need to make here. Can you provide an example?